packages/cpp/src/browser/cpp-build-configurations.ts

```diff
@@ -159,9 +168,12 @@ export class CppBuildConfigurationChanger implements QuickOpenModel {
 
         // Add one item per build config.
         configurations.forEach(config => {
+            const uri = new URI(config.directory);
             items.push(new QuickOpenItem({
-                label: config.name + (config === active ? ' ✔' : ''),
-                detail: config.directory,
+                label: config.name,
+                // add a icon for active build config, and empty placeholder for others
```

Comment:
"an icon" :)

Reply:
thanks @simark, I'll update the documentation in my follow up PR :)